### PR TITLE
fix circular import to allow importing models and metrics directly

### PR DESCRIPTION
**Pull request by safooray**
Previously if you tried importing model classes (models.py) or metric classes (metrics.py) directly in your third party code, you would get a circular import error. This PR reorganizes some files to fix that issue.

### CODE REVIEW

File: eureka_ml_insights/configs/model_configs.py [hunk 1]
```diff
@@ -21,6 +22,10 @@
 # in the secret_key_params dictionary. OR you can provide the key name and key vault URL to fetch the key from Azure Key Vault.
 # You don't need to provide both the key_vault_url and local_keys_path. You can provide one of them based on your setup.
 
+
+# Test model
+TEST_MODEL_CONFIG = ModelConfig(TestModel, {})
```
Comment (lalo) [inline eureka_ml_insights/configs/model_configs.py:27]: nit: why can't this be defined under the tests directory?

File: eureka_ml_insights/models/models.py [hunk 2]
```diff
@@ -972,3 +974,21 @@ def get_response(self, request):
 
     def handle_request_error(self, e):
         return False
+
+
+@dataclass
+class TestModel(Model):
```
[the remark this answers is not in the document]
Comment (safooray) [reply eureka_ml_insights/models/models.py:980]: So I put this here and its corresponding model config in model_configs.py for the purpose of testing the import still works from the main.py entry point with the changes I made.

[the remark this answers is not in the document]
Comment (lalo) [reply eureka_ml_insights/models/models.py:980]: not sure I follow - but couldn't another approach be to create another test file under tests that imports a subset (making sure there's no circular import) and declares this TestModel to exercise whatever code you want to stub out?

[the remark this answers is not in the document]
Comment (safooray) [reply eureka_ml_insights/models/models.py:980]: I meant I wanted to be able to run "python main.py ...." with this model config

**Review by vidhishanair — APPROVED** [review]
LGTM - ok with what you and Eduardo decide about where TestModel should go

**Review by lalo — APPROVED** [review]

**Pull request merged**